modules/apps/journal/journal-service/src/main/java/com/liferay/journal/service/impl/JournalArticleLocalServiceImpl.java

```diff
@@ -8405,6 +8405,24 @@ else if (action.equals("update")) {
 		subscriptionSender.addPersistedSubscribers(
 			JournalFolder.class.getName(), article.getGroupId());
 
+		long liveGroupId = 0;
+
+		try {
+			Group group = serviceContext.getScopeGroup();
+
+			if (group.isStagingGroup()) {
+				liveGroupId = staging.getLiveGroupId(
+					serviceContext.getScopeGroupId());
+			}
+		}
+		catch (Exception e) {
+		}
+
+		if (liveGroupId > 0) {
```

Comment:
I don't like adding staging specific things to the subscriptions API. Also, internally, both staging and regular subscriptions seem to be handled in the same way. I think we don't need to make a distinction, calling addPersistedSubscribers() should work as well.